src/utils/randomNum.js

```diff
@@ -0,0 +1,17 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+
+export const randomNum = () => {
```

Comment:
해당 파일에서 `randomNum` 함수 하나만 `export` 하는거라면, `default`를 사용하시는 것이 더 좋아보여요!

```suggestion
export default const randomNum = () => {
```
보통 여러개를 내보낼 때는 `export`를 사용하고 파일 하나에 한개만 내보내는 것이라면 `export default`를 사용하는 것 같아요.

이렇게 하면 import할 때 `import { userInput } from ...` 대신 `import userInput from ...` 으로 사용할 수 있어요!

Reply:
맞습니다! 리팩토링해서 좀더 직관적으로 보이도록 진행해야겠군요! 👍👍👍